src/components/apartment/ApartmentRow.tsx

```diff
@@ -69,18 +62,29 @@ const ApartmentRow = ({ apartment, ownershipType, lotteryCompleted, project }: I
         return `${reservation.queue_position}.`;
       };
 
+      const renderCustomerProfile = (
+        profile: ApartmentReservationCustomer['primary_profile' | 'secondary_profile']
+      ) => {
+        return (
+          <>
+            {profile?.last_name}, {profile?.first_name} {isLotteryResult && profile?.email && `\xa0 ${profile.email}`}
+          </>
+        );
+      };
+
       return (
         <div className={cx(styles.customer, isLotteryResult && styles.isLottery)}>
-          <Link to={`/${ROUTES.CUSTOMERS}/${reservation.customer || 0}`} className={styles.customerLink}>
-            {sortedApplicants.map((applicant, index) => (
-              <div key={index}>
-                {isLotteryResult && (
-                  <span className={styles.queueNumberSpacer}>{index === 0 && renderPositionNumber()}</span>
-                )}
-                {applicant.last_name}, {applicant.first_name}{' '}
-                {isLotteryResult && applicant.email && `\xa0 ${applicant.email}`}
+          <Link to={`/${ROUTES.CUSTOMERS}/${reservation.customer.id || 0}`} className={styles.customerLink}>
```

Comment:
Hmm, why is that ` || 0` needed, I don't think `customer.id` can ever be falsy?

Reply:
Yep, I think this is just some legacy check that has been there for a while. I can remove it 👍